frontend/app/email/confirm/page.tsx

```diff
@@ -0,0 +1,49 @@
+'use client'
+import {
+  Box,
+  Button,
+  Center,
+  Container,
+  FormControl,
+  Heading,
+  Input,
+  Link,
+} from '@chakra-ui/react'
+import NextLink from 'next/link'
+
+export default function EmailChangeConfirmPage() {
+  return (
+    <Center padding="60px 96px" bg={'gray.200'}>
+      <Container padding="60px 96px" bg={'white'}>
+        <Heading fontSize="2xl" mb="60px" textAlign="center" fontWeight="bold">
+          認証コードを入力
+        </Heading>
+
+        <Box mb="28px">
+          新しいメールアドレス宛てに送信した認証コードを入力してください。
```

Comment:
```suggestion
          新しいメールアドレス宛に送信した認証コードを入力してください。
```

て
ってふりがなに
つくんでしたっけ？